merge_requests.go

```diff
@@ -286,12 +299,21 @@ func (s *MergeRequestsService) UpdateMergeRequest(
 // http://doc.gitlab.com/ce/api/merge_requests.html#accept-mr
 func (s *MergeRequestsService) AcceptMergeRequest(
 	pid interface{},
-	mergeRequest int) (*MergeRequest, *Response, error) {
+	mergeRequest int,
+	opt *AcceptMergeRequestOptions) (*MergeRequest, *Response, error) {
 	project, err := parseID(pid)
 	if err != nil {
 		return nil, nil, err
 	}
-	u := fmt.Sprintf("projects/%s/merge_request/%d/merge", url.QueryEscape(project), mergeRequest)
+
+	v, err := query.Values(opt)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	//For some reason putting struct to body doesn't work
+	u := fmt.Sprintf("projects/%s/merge_request/%d/merge?%s", url.QueryEscape(project), mergeRequest, v.Encode())
```

Comment:
Could you explain what is not working? As this is done for all other calls successfully, I wonder if it's a bug in the Gitlab API? And if so, maybe it's already fixed in the latest version(s)?